docs/content/en/docs/implementing/day-2-operations/_index.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Day 2 Operations with Keptn
+description: How to operate and maintain your Keptn Apps
+weight: 500
+---
+
+After you have successfully rolled out your application by following
+the instructions in the [integration guide](../integrate),
+Keptn also assists you with day 2 operations for your application.
+
+Tasks that fall under this category include:
+
+* Updating the version of one or more workloads that are part of
+the same application.
+* Adding a new workload to an existing application
+* Monitoring the health of your application using `KeptnMetrics`
```

Comment:
should these last two point to the guides we already have [here](https://lifecycle.keptn.sh/docs/implementing/evaluatemetrics/)? 🤔 it contains metrics and hpa example

Reply:
yes good idea, will add the links to those sections